test/worker_test.rb

```diff
@@ -2,29 +2,103 @@
 
 module ToolingInvoker
   class WorkerTest < Minitest::Test
-    def test_flow
-      job_id = "123"
-      language = "ruby"
-      exercise = "bob"
-      s3_uri = "s3://..."
-      container_version = "v1"
-      execution_timeout = 10
+    def setup
+      super
 
+      @job_id = "123"
```

Comment:
I've added tests to verify the representer and analyzer flows. I've extracted some of the common code to the `setup` method. Let me know if that is the right thing to do.

Reply:
Yeah - that makes sense 🙂